src/main/java/com/example/moviemanager/service/tmdbmovieservice/model/MovieDetails.java

```diff
@@ -0,0 +1,7 @@
+package com.example.moviemanager.service.tmdbmovieservice.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public record MovieDetails(String overview, double vote_average) {
```

Comment:
vote_average is not named according to Java naming conventions. If you need to provide different naming for json property than it's variable name you can specify it with annotation @JsonProperty("vote_average")